<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>当前模式/</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDate">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0000/00/00</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHour">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>25</y>
     <width>49</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/timebkg.png);
font: 30pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMinute">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>25</y>
     <width>49</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/timebkg.png);
font: 30pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1024</width>
     <height>520</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
